profile_library/yeelight/MJDP09YL/model.json

```diff
@@ -0,0 +1,14 @@
+{
+  "calculation_strategy": "lut",
+  "created_at": "2024-10-08T08:21:17.833290",
+  "measure_description": "Measured with utils/measure script",
+  "measure_device": "TP-Link Tapo P110",
+  "measure_method": "script",
+  "measure_settings": {
+    "SAMPLE_COUNT": 3,
+    "SLEEP_TIME": 5,
+    "VERSION": "v1.14.10:docker"
+  },
+  "name": "Mijia MJDP09YL",
+  "standby_power": 0.05
```

Comment:
Were you able to measure this?
I have never seen a light consuming so little standby power.
Zigbee lights are typically 0.2-0.4W
Wifi 0.5-0.8

But this is BLE light so it could be.

Reply:
Yes. It's measured automatically. I'll check once more.